<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTransactionBuyThingEdit</class>
 <widget class="QDialog" name="FTransactionBuyThingEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transaction edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../res/hacc.qrc">
    <normaloff>:/items/transaction.png</normaloff>:/items/transaction.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Thing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ui::selector::WThingSelectButton" name="pwThingSelector" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lMoney">
     <property name="text">
      <string>Money</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="leMoney">
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbClearMoney">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/hacc.qrc">
         <normaloff>:/main/icons/edit-clear-locationbar-rtl.png</normaloff>:/main/icons/edit-clear-locationbar-rtl.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lAmount">
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="leAmount">
       <property name="text">
        <string>0.00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbClearAmount">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/hacc.qrc">
         <normaloff>:/main/icons/edit-clear-locationbar-rtl.png</normaloff>:/main/icons/edit-clear-locationbar-rtl.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lDButtons">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOk">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/hacc.qrc">
         <normaloff>:/main/icons/dialog-ok-apply.png</normaloff>:/main/icons/dialog-ok-apply.png</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../../res/hacc.qrc">
         <normaloff>:/main/icons/dialog-cancel.png</normaloff>:/main/icons/dialog-cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ui::selector::WThingSelectButton</class>
   <extends>QWidget</extends>
   <header>wthingselectbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pwThingSelector</tabstop>
  <tabstop>leMoney</tabstop>
  <tabstop>leAmount</tabstop>
  <tabstop>tbClearMoney</tabstop>
  <tabstop>tbClearAmount</tabstop>
 </tabstops>
 <resources>
  <include location="../../../res/hacc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>FTransactionBuyThingEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbOk</sender>
   <signal>clicked()</signal>
   <receiver>FTransactionBuyThingEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>42</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
